examples/umbrella_sampling/.gitignore

```diff
@@ -0,0 +1,4 @@
+start.gsd
+umbrella.gsd
+umbrella.dat
+hist.pdf
```

Comment:
Could this be placed in the main gitignore as `examples/*.gsd`, `examples/*.dat`, `examples/*.pdf`?

Reply:
sure, I put them in the global .gitignore noe.
And have it ignore all sub directories of example.